test/e2e/tests/account/import-flow.spec.js

```diff
@@ -377,7 +377,7 @@ describe('Import flow @no-mmi', function () {
     );
   });
 
-  it('Connects to a Hardware wallet for trezor', async function () {
+  it('Connects to a Hardware wallet for lattice', async function () {
     if (process.env.ENABLE_MV3) {
```

Comment:
It looks like this replaces the existing test for Trezor with a test for GridPlus Lattice. Is there a reason to not keep testing for the Trezor-connect case?